package.json

```diff
@@ -6,6 +6,7 @@
   "license": "ISC",
   "scripts": {
     "lint": "standard",
+    "dev": "nodemon node service.js",
```

Comment:
Why is it `nodemon node service.js` other than `nodemon service.js`?

Reply:
oh.. yea, i forgot about that... :D